src/com/intellij/idea/plugin/hybris/project/configurators/impl/RegularContentRootConfigurator.java

```diff
@@ -475,7 +446,7 @@ protected void configureWebInf(
             excludeDirectory(contentEntry, new File(rootDirectory, WEB_INF_CLASSES_DIRECTORY));
         } else if (
             !moduleDescriptor.getRootProjectDescriptor().isImportOotbModulesInReadOnlyMode() &&
```

Comment:
```
xxx
&& xxx
```

Reply:
will be refactored later, in scope of migrating class to Kotlin